package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "zarf-ui",
+  "private": true,
+  "version": "0.0.1",
```

Comment:
I think it's worth exploring--but we cut the release version on release like we do with zarf agent, so it will always be wrong if we try to match version.  Might be better to just keep it separate for now.